app/src/main/java/com/architectcoders/aacboard/model/Repository.kt

```diff
@@ -0,0 +1,12 @@
+package com.architectcoders.aacboard.model
+
+class Repository {
+
+    suspend fun searchPictograms(searchString: String):List<DomainPictogram> {
+        val arasaacPictograms = RemoteConnection.service.searchPictos(searchString)
+        return arasaacPictograms.map {
+            DomainPictogram(it.keywords.first().keyword, it.getUrl())
```

Comment:
Possibility of creating an extension function of ArasaacPictogram to map it to domain model and leave the repo more clean